scripts/validate

```diff
@@ -35,3 +35,17 @@ done
 
 echo "==> Checking basic shell validation"
 find ../dist -name \*.sh -exec sh -n {} \;
+
+
+echo "==> Add link checker to verify all links in dist are pointing to latest versions"
+
+for LINK_PREFIX in `find ../dist/ -type l -exec basename {} \; | grep sh | sed 's/\.sh//' | sort -V`
+do
+    if [[ `find ../dist/ -type f -name "$LINK_PREFIX*" -exec basename {} \; | sort -V | tail -1` == `readlink ../dist/$LINK_PREFIX.sh` ]]
+    then
+        echo "$LINK_PREFIX.sh is pointing to the latest `readlink ../dist/$LINK_PREFIX.sh`"
```

Comment:
These scripts generally only output errors, but I think it's a good idea to output these as a "sanity check".
Could we prefix it with `[INF]` though?

Reply:
Done